docker/test/integration/minifi/core/KafkaBrokerContainer.py

```diff
@@ -19,15 +19,15 @@ def deploy(self):
             detach=True,
             name='kafka-broker',
             network=self.network.name,
-            ports={'9092/tcp': 9092, '29092/tcp': 29092, '9093/tcp': 9093, '29093/tcp': 29093},
+            ports={'9092/tcp': 9092, '29092/tcp': 29092, '9093/tcp': 9093, '29093/tcp': 29093, '9094/tcp': 9094, '29094/tcp': 29094},
```

Comment:
It was missing, only it did not have any impact because we did not have any operations on that port from the host machine. I added it in 98ed8e2450436c2502a71f0bc99301fa4576db2c